DSPs/009-Internal-Audit.md

```diff
@@ -0,0 +1,28 @@
+# Internal Audit 
+
+**Author(s)**: @shresth31
+
+## Summary
+Create a knowledge base & checklist for dOrg's internal audits.
+
+## Why should we fund this?
+
+As dOrg is growing in number, with builders of varied levels of experties, and as the defi / smart-contract space is expanding with new and complex architectures, it's hard for all the builders to keep up with all new vulnerabilities in each project. 
+
+## Anything else you'd like to add?
+
+* The **Knowledge Base** will be structured like a blog. Every article would precisely describe a bug/ vulnerability with valid examples to repicate, ways to mitigate, and external resources to read more about it.
+* The **Checklist** would be a form. This form based on the key features of the smart-contract would sugest the dev to justify security against certain vulnerabilities. 
```

Comment:
Maybe we could pin this checklist to the audits channel?

Reply:
Definitely, we could highly recommend all the devs to fill the checklist before doing any mainnet release!